stixpy/vis/aia_reprojection.py

```diff
@@ -0,0 +1,235 @@
+from datetime import datetime
+from stixcore.ephemeris.manager import Position
+import stixcore.data.test
+from pathlib import Path
+import sunpy
+import sys
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from sunpy.map import Map
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from sunpy.coordinates import frames
+from astropy.wcs import WCS
+from pathlib import Path
+from reproject import reproject_interp
+import matplotlib.pyplot as plt
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+plt.rcParams.update({'font.size': 7})
+
+def get_SOLO_Pos(start_day, end_day):
```

Comment:
should probably use astropy Times here instead of strings or at least use sunpy's time parsing